capstone/src/main/webapp/script.js

```diff
@@ -1,28 +1,145 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function youtube() {
+  const id = document.getElementById('channelIdForAnalysis').value;
+  if (!id) {
+    return;
+  }
+  const response = await fetch('/youtube_servlet?channelId=' + id,);
+  const status = await response.json();
+  const comments = status;
+  const commentListElement = document.getElementById('comment-list');
+  commentListElement.innerHTML = '';
+  const att = await submitInput()
+  commentListElement.append(att);;
+  //commentListElement.append(att);
+  //for(let attribute in att){
+ //for (const attribute of att){
+   for(var i=0;i<att.length;i++){
+     const total =[];
+     //commentListElement.append("'" +  att[i] + "'");
+  for (let item in comments.items) {
+    const score = await callPerspective(comments.items[item].snippet.topLevelComment.snippet.textOriginal, "en", [att[i]]);
+    //commentListElement.appendChild(document.createElement('br'));
+      total.push(score);
+  }
+    totalSum = arrSum(total)
+    totalAvg = totalSum/comments.items.length;
+    commentListElement.append("AVERAGE " + att[i] + " : " + totalAvg);
+    commentListElement.appendChild(document.createElement('br'));
+  }
+  }
```

Comment:
There seems to be inconsistent spacing from around lines 13 to 29. 

Also, I suggest removing the current comments (they are remnants of old code) and describing what each part of the code does, because at the moment it's somewhat unclear what `att` is and why we have `comments.items[item].snippet.topLevelComment.snippet.textOriginal`.

Reply:
addressed in "rename JS file and add descriptions"